### DISCUSSION TITLE
Is there a shortcut to see this rectangular popup with numbers of info?

### DISCUSSION THREAD

**Discussion by jazei-AR** [ImageMagick/ImageMagick#7288] [General]
![captura](https://github.com/ImageMagick/ImageMagick/assets/72522918/b60a96ff-96ef-454d-bb4c-6cd9063c7cf8)

**Comment by snibgo** [comment]
I'm not sure what you are asking. At the console, we can issue the command:
```
magick in.png -format %P%O info:
```
A typical output will be:
```
267x233+0+0
```
For the meaning of %P and %O, see https://imagemagick.org/script/escape.php

**Comment by fmw42** [comment]
Have you tried (left or right) mouse click or click and hold? Or with option/command keys held down?

**Comment by ferdnyc** [comment]
That box will show up...

1. In that form (`width×height+X+Y`), corresponding to the current visible region of an image larger than the window, when...
    1. You drag the viewing area rectangle inside the overview window, to move the visible region of the main window.
    2. You scroll inside either the overview or the main window, to move the visible region of an image that's larger than the main window.
1. In an abbreviated `+X+Y` form that tracks the mouse position, whenever you activate a mode that permits an area selection.
1. Transitioning into the full `width×height+X+Y` form, once you've clicked and started dragging a selection rectangle from a mouse-selection mode.

Mouse-selection modes are things like:
* Edit > Cut
* Edit > Copy
* Transform > Crop
* Image Edit > Region of Interest
 
there are probably some others.

Certain selection modes, like Edit > Cut and Edit > Copy, can be activated with standard keyboard shortcuts (<kbd>Ctrl</kbd>+<kbd>X</kbd>, <kbd>Ctrl</kbd>+<kbd>C</kbd>).

But I don't know of any shortcuts or mouse gestures that will pop up the current-visible-region display when it's not being moved.

**Reply by fmw42** [reply]
Thanks for the clarification.  On the Mac, the full X11 is no longer available (only Quartz), so I did not remember exactly what activated it.
